xla/service/gpu/fusions/triton/triton_support_test.cc

```diff
@@ -271,6 +271,9 @@ using UnaryElementwiseTest = TritonSupportTestWithTypeAndOpcodeAndDeviceParam;
 
 TEST_P(UnaryElementwiseTest, IsTritonSupportedUnaryElementwise) {
   auto [data_type, opcode, cc] = GetParam();
+  if (opcode == HloOpcode::kCbrt && primitive_util::IsComplexType(data_type)) {
```

Comment:
That's actually not the best fix here! We have a filter above for HLOs that are not supported in general, such that we don't need to modify the test bodies---see `DoesOpSupportType` above. Could you please move this filter there instead?

Reply:
Done